src/hooks/useSidePanelDisplayStatus.tsx

```diff
@@ -10,28 +8,21 @@ import useResponsiveLayout from './useResponsiveLayout';
  */
 function useSidePanelDisplayStatus() {
     const {isExtraLargeScreenWidth, shouldUseNarrowLayout} = useResponsiveLayout();
-    const {preferredLocale} = useLocalize();
     const [sidePanelNVP] = useOnyx(ONYXKEYS.NVP_SIDE_PANEL, {canBeMissing: true});
-    const [isModalCenteredVisible = false] = useOnyx(ONYXKEYS.MODAL, {
-        canBeMissing: true,
-        selector: isModalCenteredVisibleSelector,
-    });
+    const {isProduction} = useEnvironment();
 
-    const isLanguageUnsupported = preferredLocale !== CONST.LOCALES.EN;
     const isSidePanelVisible = isExtraLargeScreenWidth ? sidePanelNVP?.open : sidePanelNVP?.openNarrowScreen;
 
     // The Side Panel is hidden when:
     // - NVP is not set or it is false
-    // - language is unsupported
-    // - modal centered is visible
-    const shouldHideSidePanel = !isSidePanelVisible || isLanguageUnsupported || isModalCenteredVisible || !sidePanelNVP;
-    const isSidePanelHiddenOrLargeScreen = !isSidePanelVisible || isLanguageUnsupported || isExtraLargeScreenWidth || !sidePanelNVP;
+    // - Production environment
```

Comment:
I'm confused about why this is hidden on production? Is it acting as a kind of beta? Maybe we should document the why in this comment.

Reply:
It's because of this [decision](https://expensify.slack.com/archives/C090P2M6DBL/p1764718045069619?thread_ts=1764619030.436299&cid=C090P2M6DBL) I'll add a comment!